2.1.0/docker-entrypoint.sh

```diff
@@ -15,63 +15,64 @@ set -e
 
 # first arg is `-something` or `+something`
 if [ "${1#-}" != "$1" ] || [ "${1#+}" != "$1" ]; then
-	set -- /opt/couchdb/bin/couchdb "$@"
+    set -- /opt/couchdb/bin/couchdb "$@"
 fi
 
 # first arg is the bare word `couchdb`
 if [ "$1" = 'couchdb' ]; then
-	shift
-	set -- /opt/couchdb/bin/couchdb "$@"
+    shift
+    set -- /opt/couchdb/bin/couchdb "$@"
 fi
 
 if [ "$1" = '/opt/couchdb/bin/couchdb' ]; then
-	# we need to set the permissions here because docker mounts volumes as root
-	chown -R couchdb:couchdb /opt/couchdb
+    # we need to set the permissions here because docker mounts volumes as root
+    chown -R couchdb:couchdb /opt/couchdb
 
-	chmod -R 0770 /opt/couchdb/data
+    chmod -R 0770 /opt/couchdb/data
 
-	chmod 664 /opt/couchdb/etc/*.ini
-	chmod 664 /opt/couchdb/etc/local.d/*.ini
-	chmod 775 /opt/couchdb/etc/*.d
+    chmod 664 /opt/couchdb/etc/*.ini
+    chmod 664 /opt/couchdb/etc/local.d/*.ini
+    chmod 775 /opt/couchdb/etc/*.d
 
-	if [ ! -z "$NODENAME" ] && ! grep "couchdb@" /opt/couchdb/etc/vm.args; then
-		echo "-name couchdb@$NODENAME" >> /opt/couchdb/etc/vm.args
-    else
+    if [ -z "$NODENAME" ]; then
         NODENAME=$(hostname -f)
+    fi
+
+    if [ ! -z "$NODENAME" ] && ! grep "couchdb@" /opt/couchdb/etc/vm.args; then
```

Comment:
I guess you can drop `[ ! -z "$NODENAME" ]` since we just set NODENAME :).

Reply:
Well, I thought on the case that the `hostname` command returns nothing.

Reply:
Ah, ok.

I think `-n` is more canonical than `[! -z]` :)

Reply:
Please stop using `if ! -z` :)

Should this have the `if ! grep $COOKIE` logic to prevent against duplicate entries?